README.md

```diff
@@ -3,29 +3,40 @@
 [![Forks](https://img.shields.io/github/forks/Airblader/i3.svg)](https://github.com/Airblader/i3/network)
 [![Stars](https://img.shields.io/github/stars/Airblader/i3.svg)](https://github.com/Airblader/i3/stargazers)
 
-# i3-gaps
+# Rounded i3-gaps
 
 ## What is i3-gaps?
 
+Rounded i3-gaps is a fork of i3-gaps that adds rounding.
```

Comment:
Probably should be more specific then just "rounding". Something about rounding the corners of windows